collect_app/src/main/java/org/odk/collect/android/views/SlidingTabLayout.java

```diff
@@ -203,7 +204,14 @@ private void populateTabStrip() {
             }
 
             if (mTitleFontColor != -1) {
-                tabTitleView.setTextColor(getResources().getColor(mTitleFontColor));
+
+                int tabTitleTextColor;
```

Comment:
@lognaturel  I have fixed it and once again sorry for the mess up.I am relatively new to open source.